mlflow/server/js/src/experiment-tracking/components/artifact-view-components/ShowArtifactLoggedModelView.js

```diff
@@ -195,6 +207,9 @@ class ShowArtifactLoggedModelView extends Component {
   }
 
   renderPyfuncCodeSnippet() {
+    if (this.state.loader_module === 'mlflow.spark') {
+      return this.renderMlflowSparkCodeSnippet();
```

Comment:
Can we also indicate that users can reload the model via `mlflow.pyfunc.load_model()` and perform single-node predictions via pyfunc predict (https://github.com/mlflow/mlflow/blob/97506b50fb32d0499a8239b69a10bc4812525f84/mlflow/spark.py#L896-L925)?